.circleci/config.yml

```diff
@@ -1823,26 +1843,37 @@ jobs:
           capture_line_counts: true
           cache_key_name: "agent_2-growing_logs-monitors-multiprocess-2-worker-py-35"
 
-  benchmarks-loaded-agent-single-growing-log-file-180mb-py-27:
+  benchmarks-loaded-agent-single-50mb-log-file-with-500k-lines-py-27:
```

Comment:
Just curious why this benchmark setup was changed?

Reply:
EDIT: Looks like this may be due to merge conflict resolution or similar, right?

Reply:
Yeah, sorry. I haven't sync the target branch with mater.  Please see this PR https://github.com/scalyr/scalyr-agent-2/pull/788